### ISSUE TITLE
LC0 has a bad time management when playing with "increment" 

### ISSUE THREAD

**Issue by all12** [glinscott/leela-chess#542]
LC0 has a bad time management when playing with "increment".
Tested with 10, 15, 20, 30, 40, 45 and 60 seconds per move.
After some moves LC0 got +5 to +20 minutes on clock while playing with increment!!!
The opponent got only some seconds to 2 minutes on clock while playing with increment. But also got much better position for the this little time gain.

The conclusion is that LC0 should use much more time per move when playing with increment.
GPU + NN
(Of course not when there is an easy move or only move).

**Comment by ganeshkrishnan1** [comment]
My Leela fork on Lichess (https://lichess.org/@/AiHelloBot) plays ultra bullet with time increments and I have not usually seen any issues with time management apart from network lag.

Do you have game examples with such issues?

**Comment by mooskagh** [comment]
Are you talking about openCL version from [here](https://github.com/glinscott/leela-chess/releases) or cuDNN version?

**Comment by all12** [comment]
openCL

**Comment by killerducky** [comment]
@jjoshua2 does your PR help with this?

**Comment by jjoshua2** [comment]
Yes, it should. Big elo gains with 2+2 and 10+10. I didn't try anything
crazy like 90+1 minute though

**Comment by ASilver** [comment]
Afte 300 games played with v10 at 1m+1s (Leela vs Leela), I have not seen a single loss on time, even when it is playing with less than a second left on the clock.
